str/data_frame_creator.py

```diff
@@ -191,14 +191,89 @@ def gangstr_tsv_writer(input_dir):
     return tsv
 
 
+def hipstr_csv_writer(input_dir):
+    """Creates a CSV file containing dataframe of merged HipSTR VCFs"""
+    files = to_path(input_dir).glob('*.vcf.gz')  # HipSTR output is a merged vcf.gz file
+    csv = (
+        ','.join(
+            [
+                'sample_id',
+                'h_chr',
+                'h_ref',
+                'h_pos',
+                'h_id',
+                'h_start',
+                'h_end',
+                'h_period',
+                'h_gb',
+                'h_q',
+            ]
+        )
+        + '\n'
+    )
+    for file in files:
+        if isinstance(file, GSPath):
+            file.copy(file.name)
+            file = file.name
+        reader = VCFReader(file)
+        for variant in reader:
+            chr = variant.CHROM
+            ref = variant.REF
+            pos = variant.POS
+            id = variant.ID
+            start = variant.INFO.get('START')
+            end = variant.INFO.get('END')
+            period = variant.INFO.get('PERIOD')
+            samples_dict = {}
+            i = 0
+            while i < len(reader.samples):
+                sample = reader.samples[i]
+                gb = variant.format('GB')[i]
+                q = variant.format('Q')[i][0]
+                samples_dict[sample] = (gb, q)
+                i += 1
+            for sample in samples_dict:
+                gb, q = samples_dict[sample]
```

Comment:
I think the linter's original complaint is here. Instead of using 

```
for sample in dict:
    item = dict[sample]...
```

python has `items`, `values`, and `keys` as dictionary operations, e.g.

```
for sample, content in dict.items():
    gq = content[gq]
```